reusable_components_nlp/README.md

```diff
@@ -0,0 +1,989 @@
+
+# End-to-end Reusable ML Pipeline with Seldon and Kubeflow
+
+In this example we showcase how to build re-usable components to build an ML pipeline that can be trained and deployed at scale.
+
+We will automate content moderation on the Reddit comments in /r/science building a machine learning NLP model with the following components:
+
+![](img/completed-pipeline-deploy.jpg)
```

Comment:
I don't think I understand what you mean by being difficult to see the components. Is it because the photo is long and doesn't fit in the screen? The sections below provide cropped versions of the image to show the relevant pieces whilst this picture is intended to provide a full overview.

Reply:
The wording of the previous sentence makes it seem like I should be paying attention to the list of components, which I can't read in the image. Referring to it as a preview instead would be fine.